MonkeyBot/Initializer.cs

```diff
@@ -28,6 +28,16 @@ public static async Task<IServiceProvider> InitializeAsync(ApplicationArguments
 
             DiscordSocketClient client = services.GetService<DiscordSocketClient>();
             string? token = (await DiscordClientConfiguration.LoadAsync().ConfigureAwait(false)).Token;
+            try
```

Comment:
Since I was digging in the Discord API anyway I decided to just update this in the meantime ;)